docs/docs/components/selectors.mdx

```diff
@@ -391,3 +117,224 @@ Selectors predict a bot response from a set of candidate responses.
         }
     }
     ```
+
+
+
+* **Description**
```

Comment:
This diff got incredibly messed up, but basically i just took the chunk daksh added to the end about parsing the output, and added it to the output example, and git got confused